client/app/components/DataGrid/OperatorsGrid.tsx

```diff
@@ -0,0 +1,46 @@
+"use client";
+import * as React from "react";
+import { GridRowsProp, GridColDef } from "@mui/x-data-grid";
+import Link from "next/link";
+import { Button } from "@mui/material";
+import DataGrid from "./DataGrid";
+
+// Can't pass functions from server components to client ones, so needed to create this client component to pass the columns array. The rows array doesn't contain functions, so that can be passed in the operations page server component.
+const columns: GridColDef[] = [
+  { field: "operation_id", headerName: "Operation ID", width: 150 },
+  { field: "name", headerName: "Operation", width: 150 },
+  { field: "registration_year", headerName: "Registration Year", width: 150 },
```

Comment:
Both the wireframes are models are going to change; for now I just went with what exists in both